src/controllers/asteroid-controller.js

```diff
@@ -0,0 +1,21 @@
+const { mapQueryToMeteorRequest } = require('../utils');
+const { asteroidService } = require('../services');
+
+const handleError = (res, error) => {
```

Comment:
It is a bad practice to return 500 error if we can handle it on server side - e.g. when get 404 on NASA endpoint we can return 404 to client as well

Reply:
Thank you. Error handler middleware added and it handles such cases.